deploy/load-test/stack/app.env

```diff
@@ -0,0 +1,23 @@
+APP_ENVIRONMENT=prod
+APP_KERNEL_SECRET=ThisTokenIsNotSoSecretChangeIt
+APP_WAIT_FOR=unix:///var/run/proxysql/proxysql.sock,mysql-chat:3306,mysql-connect-four-1:3306,mysql-connect-four-2:3306,mysql-connect-four-3:3306,mysql-connect-four-4:3306,mysql-connect-four-5:3306,mysql-identity:3306,redis-web-interface:6379,redis-connect-four:6379,rabbit-mq:5672,nchan:81
+APP_RABBIT_MQ_DSN=amqp://guest:guest@rabbit-mq:5672?receive_method=basic_consume&qos_prefetch_count=10&heartbeat=60
+
+APP_CHAT_DOCTRINE_DBAL_URL=mysqli://root:password@mysql-chat:3306/chat?persistent=1
+APP_CHAT_RUN_MIGRATIONS=1
+
+APP_CONNECT_FOUR_DOCTRINE_DBAL_URL=mysqli://root:password@localhost?persistent=1&unix_socket=/var/run/proxysql/proxysql.sock
+APP_CONNECT_FOUR_DOCTRINE_DBAL_DATABASE=db1
+APP_CONNECT_FOUR_DOCTRINE_DBAL_SHARDS=db1,db2,db3,db4,db5
+APP_CONNECT_FOUR_RUN_MIGRATIONS=1
+APP_CONNECT_FOUR_PREDIS_CLIENT_URL=redis://redis-connect-four:6379?persistent=1
+
+APP_IDENTITY_DOCTRINE_DBAL_URL=mysqli://root:password@mysql-identity:3306/identity?persistent=1
```

Comment:
Route `identity` and `chat` also through ProxySQL. The config needs to be adapted too.